docs/source/pages/cli_usage/command_analyze.rst

```diff
@@ -68,12 +69,20 @@ Options
 
 .. option:: --skip-deps
 
-    Skip automatic dependency analysis.
+    DEPRECATED. Dependency resolution is off by default. This flag does nothing and will be removed in the next release.
+
+.. option:: --deps-depth DEPS_DEPTH
+
+    The depth of the dependency resolution. 0: disable, 1: direct dependencies, inf: all transitive dependencies. (Default: 0)
 
 .. option:: -g TEMPLATE_PATH, --template-path TEMPLATE_PATH
 
     The path to the Jinja2 html template (please make sure to use .html or .j2 extensions).
 
+.. option::  --python-venv PYTHON_VENV
+
+    The path to the Python virtual environment of the target software component.If this is set, dependency resolution must be enabled with '--deps-depth'.
```

Comment:
Missing space: `component.If`

Reply:
Fixed in https://github.com/oracle/macaron/pull/840/commits/58a8e11146ec8e6dabf2e660522fd264a55ed363